src/NRediSearch/Client.cs

```diff
@@ -84,17 +169,21 @@ public ConfiguredIndexOptions SetNoStopwords()
 
             internal void SerializeRedisArgs(List<object> args)
             {
-                SerializeRedisArgs(_options, args);
-                if (_stopwords != null && _stopwords.Length != 0)
+                SerializeRedisArgs(_options, args, _definition);
+                if (_stopwords?.Length > 0)
                 {
                     args.Add("STOPWORDS".Literal());
                     args.Add(_stopwords.Length.Boxed());
                     args.AddRange(_stopwords);
                 }
             }
 
-            internal static void SerializeRedisArgs(IndexOptions options, List<object> args)
+            internal static void SerializeRedisArgs(IndexOptions options, List<object> args, IndexDefinition definition)
             {
+                if (definition != null)
```

Comment:
this is absolutely 100% fine as written; just a C# language tip: this is semantically identical to

``` c#
definition?.SerializeRedisArgs(args);
```

Reply:
thanks